tests/FakeItEasy.Specs/CallDescriptionsInAssertionSpecs.cs

```diff
@@ -184,7 +184,7 @@ public void AssertedCallDescriptionWithCustomValueFormatter(IFoo fake, Exception
                 .x(() => fake = A.Fake<IFoo>());
 
             "And the fake has a method with a parameter that has a custom argument value formatter"
-                .See<IFoo>(foo => foo.Bar(default(HasCustomValueFormatter)));
+                .See((IFoo foo, HasCustomValueFormatter hcvf) => foo.Bar(hcvf));

```

Comment:
Again, we don't actually care if a null is passed to `Bar`. And it makes things less clear, because `See<>` no longer explicitly indicates the interface type...

Reply:
Arguably the original wasn't even that clear. It would've been better if the typeparam was a `HasCustomValueFormatter`.